rasa/shared/core/domain.py

```diff
@@ -448,6 +462,24 @@ def __init__(
         self.store_entities_as_slots = store_entities_as_slots
         self._check_domain_sanity()
 
+    @staticmethod
```

Comment:
out of scope:I find it slightly weird that the other static methods are _above_ the constructor of `Domain`. Is that just me or is it also weird for you?

Reply:
yeah I agree 😄